samples/convert-upload-images-powervs/convert-upload-images-powervs

```diff
@@ -0,0 +1,1397 @@
+    IMAGE_PATH=./$image_name
+    DOWNLOAD_IMAGE_NAME=$image_name
+  else
+    error "Unable to fetch the url"
+  fi
+}
+
+function download_image {
+  if [[ "$1" == "centos-ova" ]];then
+    log "downloading centos ova image"
+    download_url $CENTOS_OVA_URL
+    CENTOS_DOWNLOAD_OVA_IMAGE_NAME=$DOWNLOAD_IMAGE_NAME
+    CENTOS_OVA_IMAGE=$IMAGE_PATH
+    #CENTOS_OVA_IMAGE="./Centos-83.ova.gz"
+    copy_image_file $CENTOS_OVA_IMAGE $CENTOS_OVA_OBJECT_NAME
+    CENTOS_OVA_NEW_IMAGE_PATH=$IMAGE_NEW_PATH
+  elif [[ "$1" == "rhel" ]];then
+    if echo $RHEL_URL | grep -q -i  'access.cdn.redhat.com' ; then
+      log "downloading rhel image"
+      download_url $RHEL_URL
+      RHEL_IMAGE=$IMAGE_PATH
+      RHEL_DOWNLOADED_IMAGE_NAME=$DOWNLOAD_IMAGE_NAME
+      RHEL_NEW_IMAGE_PATH=$IMAGE_NEW_PATH
+      COPY_RHEL_IMAGE=1
+    fi
+  elif [[ "$1" == "rhcos" ]];then
+    download_url $RHCOS_URL
+    RHCOS_IMAGE=IMAGE_PATH
+    RHCOS_DOWNLOAD_IMAGE_NAME=$DOWNLOAD_IMAGE_NAME
+    copy_image_file $RHCOS_IMAGE $RHCOS_OBJECT_NAME
+    RHCOS_NEW_IMAGE_PATH=$IMAGE_NEW_PATH
+    COPY_RHCOS_IMAGE=1
+  elif [[ "$1" == "centos" ]];then
+    download_url $CENTOS_URL
+    CENTOS_IMAGE=$IMAGE_PATH
+    CENTOS_DOWNLOAD_IMAGE_NAME=$DOWNLOAD_IMAGE_NAME
+    copy_image_file $CENTOS_IMAGE $CENTOS_OBJECT_NAME
+    CENNTOS_NEW_IMAGE_PATH=$IMAGE_NEW_PATH
+    COPY_CENTOS_IMAGE=1
+  else
+    warn "Unknown image"
+  fi
+}
+
+function main {
+  mkdir -p ./logs
+  vars=""
+
+  # Only use sudo if not running as root
+  [ "$(id -u)" -ne 0 ] && SUDO=sudo || SUDO=""
+
+  if [[ $(uname -m) != "x86_64" ]]; then
+    warn "Only 64-bit x86 machines are supported"
+    error "Unsupported machine: $(uname -m)" 1
+  fi
+  PLATFORM=$(uname)
+  case "$PLATFORM" in
+    "Darwin")
+      PACKAGE_MANAGER="brew"
+      OS="darwin"
+      IBM_CLI_OS="osx"
+      OC_CLI_OS="mac"
+      ARCHIVE_TYPE="tar.gz"
+      ;;
+    "Linux")
+      # Linux distro, e.g "Ubuntu", "RedHatEnterpriseWorkstation", "RedHatEnterpriseServer", "CentOS", "Debian"
+      DISTRO=$(lsb_release -ds 2>/dev/null || cat /etc/*release 2>/dev/null | head -n1 || uname -om || echo "")
+      if [[ "$DISTRO" != *Ubuntu* &&  "$DISTRO" != *Red*Hat* && "$DISTRO" != *CentOS* && "$DISTRO" != *Debian* && "$DISTRO" != *RHEL* && "$DISTRO" != *Fedora* ]]; then
+        warn "Linux has only been tested on Ubuntu, RedHat, CentOS, Debian and Fedora distributions please let us know if you use this utility on other distributions"
+      fi
+      if [[ "$DISTRO" == *Ubuntu* || "$DISTRO" == *Debian*  ]]; then
+        PACKAGE_MANAGER="$SUDO apt-get"
+      elif [[ "$DISTRO" == *Fedora* ]]; then
+        PACKAGE_MANAGER="$SUDO dnf"
+      else
+        PACKAGE_MANAGER="$SUDO yum"
+      fi
+      OS="linux"
+      IBM_CLI_OS="linux64"
+      OC_CLI_OS="$OS"
+      ARCHIVE_TYPE="tar.gz"
+      ;;
+    "MINGW64"*|"CYGWIN"*|"MSYS"*)
+      PACKAGE_MANAGER=""
+      OS="windows"
+      IBM_CLI_OS="win64"
+      OC_CLI_OS="$OS"
```

Comment:
Unused. I guess you don't use ibmcloud CLI?

Reply:
Script uses cli to get the private network and the service information